tests/integration/Handlers/ConnectionTest.php

```diff
@@ -15,8 +15,6 @@ class ConnectionTest extends \Codeception\TestCase\WPTestCase {
 	public function _before() {
 
 		parent::_before();
-
-		require_once 'includes/Handlers/Connection.php';
```

Comment:
Removed unnecessary require that was causing the integration tests to fail with this error:

```
Fatal error: Cannot declare class SkyVerge\WooCommerce\Facebook\Handlers\Connection, because the name is already in use in /Users/deivamagalhaes/Code/facebook-for-woocommerce/includes/Handlers/Connection.php on line 22
```